docs/src/operations/requirements.md

```diff
@@ -41,10 +41,7 @@ find yourself in need of support.
 Running an Agave validator for live clusters (including mainnet-beta) inside Docker is
 not recommended and generally not supported. This is due to concerns of general
 Docker's containerization overhead and resultant performance degradation unless
-specially configured.
-
-We use Docker only for development purposes. Docker Hub contains images for all
-releases at [solanalabs/solana](https://hub.docker.com/r/solanalabs/solana).
+specially configured. We use Docker only for development purposes.
```

Comment:
I know this line isn't in the diff, but I think the above sentence could use a tweak:
"This is due to concerns of general Docker's ..." ==> "This is due to general concerns of Docker's ...`

Reply:
Updated